server/routes/wishCards.js

```diff
@@ -232,6 +232,20 @@ router.get('/me', renderPermissions, async (req, res) => {
   }
 });
 
+// @desc    Render createWiscard.ejs. Empty form for creating a new wish card
+// @route   GET '/wishcards/new'
+// @access  Agency
+// @tested 	Yes
+router.get('/new', renderPermissions, async (_req, res) => {
```

Comment:
Sorry for being nit picky, but can you change '/wishcards/new' to '/wiscards/create' ? 
Just because it sounds like you are viewing a new wish card instead of creating it

Reply:
yes mam